### PR TITLE
portfolio website

### PR DESCRIPTION
**Pull request by sonikaRawat**
added home page
added styles to page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (Kamba56) [inline index.html]: - [ ] Kindly use an indentation of two spaces in all your documents for readability. This is a convention and also best practice.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+<nav class="navbar">
+    <div class="logo"><a href="#">My Logo</a></div>
+    <div class="manu-icon"><img src="images/menu.png"> </div>
```
Comment (Kamba56) [inline index.html]: - [ ] kindly wrap your hamburger icon in a `button` or `a` tag. This is best for HTML semantics.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <section class="header-section">
+     <div class="div-header">   
+    <h1 class="header">I’m Mariane
+        Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#">LET’S CONNECT</a>
+    
+
+    <ul class="list">
+        <li class="list-item"><img src="images/twitter.png"> </li>
```
Comment (Kamba56) [inline index.html]: - [ ] Consider wrapping the icons in an `a` tag with `href` attribute that points to your social media handles. This improves the user experience of your design.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @sonikaRawat , 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @sonikaRawat , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**